subprojects/docs/src/docs/userguide/authoring-builds/incremental_builds.adoc

```diff
@@ -411,19 +415,17 @@ Specifically, they never can be:
 * executed in parallel with other tasks, even if <<multi_project_configuration_and_execution.adoc#sec:parallel_execution,parallel execution>> is enabled,
 * executed incrementally.
 
-The in-memory representation of the file system state (aka _<<file_system_watching.adoc#sec:daemon_watch_fs,Virtual File System>>)_ is also invalidated before an invalid task is executed.
-====
+The in-memory representation of the file system state (_<<file_system_watching.adoc#sec:daemon_watch_fs,Virtual File System>>)_ is also invalidated before an invalid task is executed.
 
 [[sec:task_input_output_runtime_api]]
-=== Runtime API
+== Runtime API
 
 Custom task classes are an easy way to bring your own build logic into the arena of incremental build, but you don’t always have that option. That's why Gradle also provides an alternative API that can be used with any tasks, which we look at next.
 
 When you don’t have access to the source for a custom task class, there is no way to add any of the annotations we covered in the previous section. Fortunately, Gradle provides a runtime API for scenarios just like that. It can also be used for ad-hoc tasks, as you'll see next.
 
-
 [[sec:runtime_api_for_adhoc]]
-==== Using it for ad-hoc tasks
+=== Managing inputs and outputs of ad-hoc tasks
```

Comment:
❌ I think this should be about "declaring", not managing.

Reply:
Done